app/components/ContactsTable.js

```diff
@@ -0,0 +1,36 @@
+import TableRow from "./TableRow";
+import RerouteBtn from "./RerouteBtn";
+import PropTypes from "prop-types";
+
+export default function ContactsTable({contacts, handleDeleteClick}) {
+
+  return(
+    <>
+      <table className="table table-hover">
+        <thead>
+          <tr>
+            <th scope="col"> Profile Pic </th>
+            <th scope="col"> Name </th>
+            <th scope="col"> Email </th>
+            <th scope="col"> Phone </th>
+            <th scope="col"></th>
```

Comment:
here is an example where missing comments help, why do you have this empty column? I have to go to the data to notice its for the delete btn